File: haskell/string/1120.hs
-- title: Revisão de Contrato
-- number of the problem: 1120
-- date: 25/09/20
-- author: ltsdw

myLoop :: [Char] -> [Char] -> IO ()
myLoop digit number = do
    l1 <- getLine 
    let (digit : number: _) = words l1
    if digit /= "0" && number /= "0"
        then do
            let correct_number = filter (/= head digit) number
            print (if correct_number /= [] then (read correct_number :: Integer) else 0)
            myLoop digit number
    else do return ()

main :: IO ()
main = myLoop "1" "1"

File: python/string/1120.py
'''
title: Revisão de Contrato
number of the problem: 1120
date: 11/03/2018
author: ltsdw
'''

l = []

while True:
    n, num = map(int, input().split())

    if not n and not num:
        break

    n = str(n)
    num = str(num)

    for _ in num:
        num = num.replace(n, '')
 
    if num == '':
        num = num.replace('', '0')
 
    l += [int(num)]

for i in l:
    print(i)

